docs/pages/User/Install/_index.md

```diff
@@ -1,24 +1,134 @@
+{{% /expand%}}
+
+{{%expand "Azure Kubernetes Service (AKS)" %}}
+
+AKS clusters have by default with the following PodCIDR and ServiceCIDR:
+
+If you are using Azure CNI:
+
+```bash
+SUBNET_ID=$(az aks list --query="[?name=='__YOUR_CLUSTER_NAME__']" | jq -r '.[0].agentPoolProfiles[0].vnetSubnetId')
+POD_CIDR=$(az network vnet subnet show --ids ${SUBNET_ID} | jq -r .addressPrefix)
+SERVICE_CIDR=$(az aks list --query="[?name=='__YOUR_CLUSTER_NAME__']" | jq -r ".[0].networkProfile.serviceCidr")
+```
+
+Or Kubenet:
+
+```bash
+POD_CIDR=$(az network vnet subnet show --ids ${SUBNET_ID} | jq -r ".[0].networkProfile.serviceCidr)
+SERVICE_CIDR=$(az aks list --query="[?name=='__YOUR_CLUSTER_NAME__']" | jq -r ".[0].networkProfile.serviceCidr"
+```
+
+
+{{% /expand%}}
+{{%expand "Google Kubernetes Engine (GKE)" %}}
+
+```bash
+SERVICE_CIDR=$(gcloud container clusters describe __YOUR_CLUSTER_NAME__ --zone -__YOUR_ZONE__ --project __YOUR_PROJECT_ID__ --format="json" | jq -r `.servicesIpv4Cidr`)
+POD_CIDR=$(gcloud container clusters describe __YOUR_CLUSTER_NAME__ --zone -__YOUR_ZONE__ --project __YOUR_PROJECT_ID__ --format="json" | jq -r `.clusterIpv4Cidr`)
+API_SERVER=$(gcloud container clusters describe __YOUR_CLUSTER_NAME__ --zone -__YOUR_ZONE__ --project __YOUR_PROJECT_ID__ --format="json" | jq -r `.endpoint`)
+```
+
+{{% /expand%}}
+{{%expand "K3s" %}}
+
+K3s clusters have by default with the following PodCIDR and ServiceCIDR:
+
+| Variable               | Default | Description                                 |
+| ---------------------- | ------- | ------------------------------------------- |
+| `networkManager.config.podCIDR`             |    10.42.0.0/16     |
+| `networkManager.config.serviceCIDR`         |    10.43.0.0/16     |
+{{% /expand%}}
+
+#### Set-Up Liqo Repository
+
+Firstly, you should add the official Liqo repository to your Helm Configuration:
+
+```bash
+helm repo add liqo https://helm.liqo.io/
+```
+
+#### Set-up
+
+The most important values you can set are the following:
+
+| Variable               | Description                                 |
+| ---------------------- | ------------------------------------------- |
+| `networkManager.config.podCIDR`        | The cluster Pod CIDR                                 |
+| `networkManager.config.serviceCIDR`    | The cluster Service CIDR                             |
+| `discovery.config.clusterLabels`       | Labels used to characterize your cluster's resources |
+| `auth.config.allowEmptyToken`          | Enable/disable [cluster pre-authentication](/User/Configure/Authentication)            |
+
+Example:
+
+```bash
+helm install liqo liqo/liqo -n liqo --create-namespace  --set networkManager.config.podCIDR="10.42.0.0/16" --set networkManager.config.serviceCIDR="10.96.0.0/12" --set discovery.config.clusterLabels.region="A" --set discovery.config.clusterLabels.foo="bar" 
+```
+
+After a couple of minutes, the installation process will be completed. You can check if Liqo is running by:
```

Comment:
Not clear to me when actually Liqo has completed the installation process.
Does this happen when the previous command returns? In this case, I would state this explicitly, e.g., "The previous command will take a couple of minutes to complete; when this happens, the installation process is completed.

Reply:
Yes, you are right it is more clear, ty.